lib/matplotlib/textpath.py

```diff
@@ -321,21 +310,13 @@ def get_glyphs_tex(self, prop, s, glyph_map=None,
             if char_id not in glyph_map:
                 font.clear()
                 font.set_size(self.FONT_SCALE, self.DPI)
-                if enc:
-                    charcode = enc.get(glyph, None)
-                else:
-                    charcode = glyph
-
-                ft2font_flag = LOAD_TARGET_LIGHT
-                if charcode is not None:
-                    glyph0 = font.load_char(charcode, flags=ft2font_flag)
+                # See comments in _get_ps_font_and_encoding.
+                if enc is not None:
+                    index = font.get_name_index(enc[glyph])
```

Comment:
this used to handle the case where `glyph not in enc` where as this looks like it will now raise `KeyError`.  Is this a change we want to make?

Reply:
Restored handling of `glyph not in enc`.  Not sure why I removed this in the first place.